vulcanai/utils.py

```diff
@@ -419,3 +430,163 @@ def stitch_datasets(df_list, on, index_list=None):
     merged_df.fillna(0, inplace=True)
 
     return merged_df
+
+
+def feature_directionality(features, data, val_x, val_y, dense_net, filename):
```

Comment:
let's throw the dense_net (let's just call it network) at the front of the inputs.